core/src/test/java/org/javarosa/core/model/test/FormDefTest.java

```diff
@@ -431,14 +432,62 @@ public void testSimilarBindConditionsAreDistinguished() throws Exception {
         }
     }
 
+
+
+    /**
+     * Regression: IText function in xpath was not properly using the current locale instead of the
+     * backup
+     */
+    @Test
+    public void testITextXPathFunction() throws XPathSyntaxException {
+        FormParseInit fpi = new FormParseInit("/xform_tests/itext_function.xml");
+        FormEntryController fec =  initFormEntry(fpi, "new");
```

Comment:
can you add a comment that says we are using the 'new' locale instead of 'old', which is the form's default?